contents/blog/equity_explainer.md

```diff
@@ -0,0 +1,50 @@
+
+Making matters more complicated, the majority of standard terms in stock option plans limit your freedom in when and how you can exercise your options. 
+
+### At PostHog, we try to do things differently:
+
+|                           | **PostHog**                 | **The Other Guys**           |
+|---------------------------|-----------------------------|------------------------------|
+| **Vesting Start:**        | On your first day           | After a "probationary period |
+| **Exercise Window:**      | 10 _years_ after departure  | 90 _days_ after departure    |
+| **Merger or Acquisition** | Double-Trigger acceleration | No acceleration              |
+
+### Vesting Start:
+
+At PostHog, we recognize that you start contributing to the growth of our company on day 1 (and even before that, on your SuperDay), so we feel that you should start earning your options on that day as well (because our lawyers won’t let us give you options until you’re actually hired). Many other start-up companies make you wait until after a probationary period, often 3-months, before your options start to vest.
+
+### Exercise Window:
+
+Regardless of whether you resign, or are let go for performance reasons, you will have 10 years from the day you were hired to exercise your options. In most companies, you have only 3 months to exercise your options before you lose them entirely, which can make tax optimization and planning a stressful and difficult process.
+
+In the unlikely event that you are let go “with cause” (AKA gross misconduct like embezzlement or discrimination), you will forfeit your options, which we feel is only fair.
+
+### Merger or Acquisition:
+
+In the event that PostHog is sold to another company before we reach an IPO, we have preferable terms that ensure team members who contributed to growing the company to that point are rewarded for their efforts. If the company is acquired (first trigger) AND you are terminated as part of that acquisition (second trigger), all of your options vest immediately and your 10-year window to exercise them begins. This can offer some protection and certainty to employees who exercise their options following this trigger, as shares in the company during acquisition are much more tangible than options to buy shares, and are often converted into shares of the acquiring company or bought back by the acquirer for cash.
+
+As a fully open-source company, we try to be as transparent as possible about compensation including equity, because we firmly believe that everyone who contributes to our success should reap the benefits. For more information about how PostHog thinks about equity, check out our compensation page and equity FAQ in our handbook.
```

Comment:
```suggestion
As an open-source company, we are transparent about compensation including equity, because we believe that everyone who contributes to our success should reap the benefits. For more information about how PostHog thinks about equity, check out our compensation page and equity FAQ in our handbook.
```